device/device.go

```diff
@@ -322,6 +325,18 @@ func NewDevice(tunDevice tun.Device, bind conn.Bind, logger *Logger) *Device {
 	return device
 }
 
+// BatchSize returns the BatchSize for the device as a whole which is the max of
+// the bind batch size and the tun batch size. The batch size reported by device
+// is the size used to construct memory pools, and is the allowed batch size for
+// the lifetime of the device.
+func (device *Device) BatchSize() int {
+	sz := device.net.bind.BatchSize()
+	if sz < device.tun.device.BatchSize() {
+		sz = device.tun.device.BatchSize()
```

Comment:
`device.tun.device.BatchSize()` is called twice, implying no side effects. I dunno what Go conventions are here, given that there's no `const` specifier like in C++. But I would think you'd assign both to variables, and then compare variables? Or does this not matter at all?

Reply:
Updated to use a variable, it's not called frequently - just at setup time, so it's not a significant concern. I'd actually hope the compiler disappears most of this most of the time.